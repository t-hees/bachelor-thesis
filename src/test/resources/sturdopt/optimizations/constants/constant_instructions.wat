(module
  (func (export "main") (result i32)
    i32.const 1
    i32.const 2
    i32.add

    i32.const 5
    i32.const 5
    i32.sub
    i32.eqz

    i32.lt_u
  )
)
