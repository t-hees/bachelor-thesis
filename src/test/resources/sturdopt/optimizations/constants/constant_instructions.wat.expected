(module
  (func (export "main") (result i32)
    i32.const 1
    i32.const 2
    drop
    drop
    i32.const 3

    i32.const 5
    i32.const 5
    drop
    drop
    i32.const 0
    drop
    i32.const 1

    drop
    drop
    i32.const 0
  )
)
